src/css/_widgets.scss

```diff
@@ -0,0 +1,54 @@
+.widget {
+  padding:30px 20px;
+  border: 2px solid #000;
+}
+
+.user-profile-widget {
+  display:flex;
+  flex-flow:column;
+  align-self: flex-start;
+  width:20%;
+  margin-right:5%;
+}
+
+.trip-widget {
+  width:75%;
+
+  .trip-filter-bar {
+    display:flex;
+    align-items: center;
+    margin-bottom:30px;
+  }
+}
+
+.traveler-trip-list {
+  .trip {
+    margin-bottom:20px;
+    display:flex;
+    align-items:center;
```

Comment:
For this part, it is a similar structure to the line of 18-20 in this file.  I believe that you can create a mixin that uses parameters and arguments to reduce the overall size of the code.   With a mixin, you can use the same mixin setup for any other sections that use the same setup for the display, margin-bottom, and align-items.

Reply:
Nice job on spotting those similarities. I'll create a mixin for that. Thanks!